### PR TITLE
Fix Linux build support by removing improbable namespace around UnrealObjectRef

### PR DESCRIPTION
**Pull request by joshuahuburn**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Fix Linux build support by removing improbable namespace around UnrealObjectRef

Please discuss if you disagree with this change.

#### Tests
Builds on Linux
#### Primary reviewers
@danielimprobable @davedolben

### CODE REVIEW

File: SpatialGDK/Source/Public/Schema/UnrealObjectRef.h [hunk 1]
```diff
@@ -63,22 +60,20 @@ struct UnrealObjectRef
...
-}
-
 // This template specialization informs THasGetTypeHash
-// that there exists a valid GetTypeHash function for improbable::UnrealObjectRef.
+// that there exists a valid GetTypeHash function for UnrealObjectRef.
 // This is required as THasGetTypeHash fails to resolve GetTypeHash functions referencing 
 // classes inside namespaces.
 template <>
-struct THasGetTypeHash<improbable::UnrealObjectRef>
+struct THasGetTypeHash<UnrealObjectRef>
```
Comment (improbable-valentyn) [inline SpatialGDK/Source/Public/Schema/UnrealObjectRef.h]: You can remove this struct as well since it was only needed because of the namespace.

**Comment by davedolben** [comment]
Seems reasonable. If we're not expecting end users to interact with this struct, then the naming shouldn't actually matter that much and we'll be able to rename it under the hood and not affect existing projects.

**Comment by m-samiec** [comment]
Not a hill I want to die on, but should we make the name more verbose? Improbable_UnrealObjRef?

**Comment by Vatyx** [comment]
I personally think UnrealObjRef, without being in blueprints for now, is a fine name.

**Comment by danielimprobable** [comment]
The `UnrealObjectRef` will be moved into the engine with my generated Object ref contexts and will become a USTRUCT so I think this PR is good to go with the current naming

**Pull request merged**